conda_forge_tick/migrators/nvtools.py

```diff
@@ -0,0 +1,244 @@
+
+        Parameters
+        ----------
+        recipe_dir : str
+            The directory of the recipe
+        attrs : dict
+            The node attributes
+
+        Returns
+        -------
+        namedtuple or bool:
+            If namedtuple continue with PR, if False scrap local folder
+        """
+        # STEP 0: Bump the build number
+        self.set_build_number(os.path.join(recipe_dir, "meta.yaml"))
+
+        # STEP 1: Add cf-nvidia-tools to build requirements
+        meta = os.path.join(recipe_dir, "meta.yaml")
+        if _file_contains(meta, "cf-nvidia-tools"):
+            logging.debug("cf-nvidia-tools already in meta.yaml; not adding again.")
+        else:
+            _insert_requirements_build(meta)
+            logging.debug("cf-nvidia-tools added to meta.yaml.")
+
+        # STEP 2: Add check-glibc to the build script
+        build = os.path.join(recipe_dir, "build.sh")
+        if os.path.isfile(build):
+            if _file_contains(build, "check-glibc"):
+                logging.debug(
+                    "build.sh already contains check-glibc; not adding again."
+                )
+            else:
+                with open(build, "a") as file:
+                    file.write('\ncheck-glibc "$PREFIX"/lib/*.so.* "$PREFIX"/bin/*\n')
+                logging.debug("Added check-glibc to build.sh")
+        else:
+            if _file_contains(meta, "check-glibc"):
+                logging.debug(
+                    "meta.yaml already contains check-glibc; not adding again."
+                )
+            else:
+                _insert_build_script(meta)
+                logging.debug("Added check-glibc to meta.yaml")
+
+        # STEP 3: Remove os_version keys from conda-forge.yml
+        config = os.path.join(recipe_dir, "..", "conda-forge.yml")
+        with open(config) as f:
+            y = yaml_safe_load(f)
+        y_orig = copy.deepcopy(y)
+        y.pop("os_version", None)
+        if y_orig != y:
+            with open(config, "w") as f:
+                yaml_safe_dump(y, f)
+
+        return self.migrator_uid(attrs)
+
+    def pr_body(
+        self, feedstock_ctx: ClonedFeedstockContext, add_label_text=True
+    ) -> str:
+        """Create a PR message body
+
+        Returns
+        -------
+        body: str
+            The body of the PR message
+            :param feedstock_ctx:
+        """
+        body = f"{AddNVIDIATools.__doc__}\n\n"
+
+        if add_label_text:
+            body += (
+                "If this PR was opened in error or needs to be updated please add "
+                "the `bot-rerun` label to this PR. The bot will close this PR and "
+                "schedule another one. If you do not have permissions to add this "
+                "label, you can use the phrase "
+                "<code>@<space/>conda-forge-admin, please rerun bot</code> "
+                "in a PR comment to have the `conda-forge-admin` add it for you.\n\n"
```

Comment:
Wonder if we should add some text on how to handle issues or ask for help in resolving

Reply:
Please look at line 113. The description directs users (us) to ping me for questions.